internal/api/external.go

```diff
@@ -531,6 +531,8 @@ func (a *API) Provider(ctx context.Context, name string, scopes string) (provide
 		return provider.NewFacebookProvider(config.External.Facebook, scopes)
 	case "notion":
 		return provider.NewNotionProvider(config.External.Notion)
+	case "supabase":
+		return provider.NewSpotifyProvider(config.External.Spotify, scopes)
```

Comment:
This seems wrong. :/

Reply:
oh lol my bad, this was a change from something else i was trying, lemme remove it